test/editor/deserialize-test.js

```diff
@@ -237,18 +237,6 @@ describe('editor/deserialize', function() {
             expect(parts[3]).toStrictEqual({ type: "newline", text: "\n" });
             expect(parts[4]).toStrictEqual({ type: "plain", text: "3. Finish" });
         });
-        it('non tight lists', () => {
```

Comment:
I removed this test because there doesn't appear to be a way to actually create non-tight lists with the current Commonmark parser, so there's no reason to deserialize them differently.